client/src/components/App.js

```diff
@@ -30,9 +31,8 @@ const App = (props) => {
     <Router>
       <TopBar user={currentUser} />
       <Switch>
-        <Route exact path="/">
-          <h2>Hello from react</h2>
-        </Route>
+        <Route exact path="/" component=
+        {HomePage} />
```

Comment:
when Routes are written spanning multiple lines, the closing tag should be on its own new line
```js
<Route
  exact path....
/>
```